metricbeat/Makefile

```diff
@@ -19,11 +19,16 @@ GOX_FLAGS=-arch="amd64 386 arm ppc64 ppc64le"
 
 include ${ES_BEATS}/libbeat/scripts/Makefile
 
+.PHONY: clean
+clean::
+	# Clean generated dashbord directories
+	rm -rf _meta/kibana/dashboard _meta/kibana/search _meta/kibana/visualization
+	# Clean module docs
+	rm -rf docs/modules
+
 # Collects all module dashboards
 .PHONY: kibana
 kibana:
```

Comment:
Hmm, yeah, still feels risky to me. Maybe separate a `kibana-clean` target on which both clean and kibana depend?